Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[workspace]
+members = [
+    "miden-proto",
+    "miden-rpc",
+    "miden-store",
+    "utils",
+]
```

Comment:
Thinking about this some more, we can probably don't need to prefix directories with `miden-`. So, this could be just:
```
members = [
    "proto",
    "rpc",
    "store",
    "utils",
]
```
For crate names, however, we should probably prefix them with `miden-node-`. For example: `miden-node-proto`, `miden-node-rpc`, `miden-node-store`, `miden-node-utils` etc.

What do you think?